x/iro/keeper/settle.go

```diff
@@ -86,8 +86,9 @@ func (k Keeper) Settle(ctx sdk.Context, rollappId, rollappIBCDenom string) error
 // - Creates a balancer pool with the determined tokens and DYM.
 // - Uses leftover tokens as incentives to the pool LP token holders.
 func (k Keeper) bootstrapLiquidityPool(ctx sdk.Context, plan types.Plan) (poolID, gaugeID uint64, err error) {
-	unallocatedTokens := plan.TotalAllocation.Amount.Sub(plan.SoldAmt)        // assumed > 0, as we enforce it in the Buy function
-	raisedDYM := k.BK.GetBalance(ctx, plan.GetAddress(), appparams.BaseDenom) // assumed > 0, as we enforce it by IRO creation fee
+	reserve := math.NewIntWithDecimal(1, int(plan.BondingCurve.SupplyDecimals()))
```

Comment:
are we guranteed that `plan.BondingCurve.SupplyDecimals` will always be = `rollapp.GenesisInfo.NativeDenom.Exponent` ? 
as the first is used to calcualte the rollapp token fee on the liquidity pool and the latter upon creation. 

the `reserve` here - same comment as before about referring it to rollapp creation fee.

Reply:
`rollapp.GenesisInfo.NativeDenom.Exponent`  supports only 18
there's TODO to add support if needed

Reply:
renamed